src/commandsAndMenu.tsx

```diff
@@ -377,6 +378,41 @@ export function addCommands(
     }
   });
 
+  /** Add git push force command */
+  commands.addCommand(CommandIDs.gitForcePush, {
```

Comment:
Let's reuse the existing `gitPush` command - I will explain how in a PR comment.